src/actions/learn/index.cr

```diff
@@ -0,0 +1,5 @@
+class Learn::Index < BrowserAction
+  get "/learn" do
+    html Learn::IndexPage, title: "Learn the Lucky Framework"
```

Comment:
How do you feel about the title of all these pages going in the code for the page?

Reply:
I think the idea Paul was going for here is that this makes the title required to be passed in to ensure you don't make a new page and forget to set the title. I think we can do it differently though.

Reply:
Yeah that's fair. I don't feel super strongly about it to be clear. I just found it odd that it's the action's responsibility to set the title for all pages. Felt like the page should own that. NBD